src/config.js

```diff
@@ -17,49 +17,49 @@ export default {
     '!drop': {
       emoteMultiplier: 1,
       velocities: {
-        min: 3,
-        max: 7,
+        min: Math.floor(Math.random() * 5) + 3,
+        max: Math.floor(Math.random() * 7) + 5,
       },
       strategy: 'drop',
     },
     '!bigdrop': {
       emoteMultiplier: 1,
       velocities: {
-        min: 3,
-        max: 7,
+        min: Math.floor(Math.random() * 5) + 3,
+        max: Math.floor(Math.random() * 7) + 5,
       },
       strategy: 'drop',
     },
     '!rain': {
       emoteMultiplier: 15,
       velocities: {
-        min: 3,
-        max: 7,
+        min: Math.floor(Math.random() * 5) + 3,
+        max: Math.floor(Math.random() * 7) + 5,
       },
       strategy: 'dropSpecificSizedPanthers',
       size: emotes.sizes[1],
     },
     '!blizzard': {
       emoteMultiplier: 15,
       velocities: {
-        min: 20,
-        max: 30,
+        min: Math.floor(Math.random() * 25) + 20,
```

Comment:
The min here is between 20 and 45
The max here is between 25 and 55

So a chance that your max could be lower than your min.